packages/components/package.json

```
@@ -22,7 +22,7 @@
     "build:esm": "ttsc --project tsconfig.build.json --module es2015 --target es5 --outDir dist/esm",
     "build:cjs": "ttsc --project tsconfig.build.json --module commonjs --target es5 --outDir dist/cjs",
     "clean": "rm -Rf ./dist",
-    "develop": "yarn clean && ttsc --project tsconfig.build.json --watch --module commonjs --target es5 --outDir dist/cjs"
+    "develop": "ttsc --project tsconfig.build.json --watch --module commonjs --target es5 --outDir dist/cjs"
```

```suggestion
    "develop": "yarn clean && ttsc --project tsconfig.build.json --watch --module commonjs --target es5 --outDir dist/cjs"
     "develop-playground": "ttsc --project tsconfig.build.json --watch --module commonjs --target es5 --outDir dist/cjs"
```

For now (until we can rationalize this to a single command) perhaps we can do this as two separate commands rather than clobbering the Gatsby-compliant version?